src/ObjectGraph.es6.js

```diff
@@ -74,6 +74,10 @@ ObjectGraph.prototype.blacklistedObjects = [
 }.bind(typeof window === 'undefined' ? global : window)).filter(
   value => value !== undefined
 );
+ObjectGraph.prototype.blacklistedTypes = [
+  'MimeType'   // causing error in some previous versions of Chrome
```

Comment:
nit: whenever possible, phrase comment as a sentence (start with capital letter, end with period). Also, please be more specific; e.g.,

"This type initiates a process of continually extending the object graphs in Chrome <= [[largest version number with issue]]."